neuromation/api/jobs.py

```diff
@@ -162,14 +171,43 @@ def __init__(self, core: _Core, config: _Config) -> None:
             # an error is raised for status >= 400
             return None  # 201 status code
 
-    async def save(self, id: str, image: RemoteImage) -> None:
+    async def save(
```

Comment:
Looking on dependency graph I'm inclining to a thought that this method actually belongs to images, not jobs.

Reply:
as far as I understood, the CLI command should be `neuro job save`, not `neuro image save` :confused:

Reply:
This is a good question.
In docker `save` belongs to `docker image` group, not `docker container`.
Maybe we should apply the same change to our CLI.

Anyway, for now please keep CLI command as is but move API.

Reply:
ok

Reply:
I moved both `save()` back to `jobs` because when it belongs to `images`, this functionality does not work on windows as `Images.__init__()` requires docker available on the local machine, which is not necessary for `save()`.

Let's keep it less clean for now, and later refactor it out to a separate class (a separate member of `Client`).

Reply:
Separate member of `Client` sounds even better.
What name do you suggest?
What prevents you from making this refactoring now?

Reply:
> What name do you suggest

maybe something like `ImagesLocal` (for current `Images`) and `ImagesRemote`? meaning if the user's machine works with local docker or remote docker.

> What prevents you from making this refactoring now?

It will require many non-functional changes; let's keep this PR as-is and do refactoring it in a separate PR?